Assets/Scripts/Game/UserInterfaceWindows/DaggerfallTradeWindow.cs

```diff
@@ -613,6 +613,12 @@ protected override void FilterRemoteItems()
                 base.FilterRemoteItems();
         }
 
+        protected void ResetWagonButton()
```

Comment:
would it have been more robust to just use ShowWagon(false)?

Reply:
I think so yeah.